core/services/ocr2/plugins/ocr2keeper/evm21/upkeepstate/store.go

```diff
@@ -111,6 +123,9 @@ func (u *upkeepStateStore) Start(pctx context.Context) error {
 			ticker := time.NewTicker(utils.WithJitter(u.cleanCadence))
 			defer ticker.Stop()
 
+			flushTicker := newTickerFn(flushCadence)
+			defer flushTicker.Stop()
```

Comment:
One request that came to me when lasting working on this was to add jitter to the tickers that were making database calls. I don't think first one needs the jitter anymore, but the second one does.